pkg/jobrunaggregator/jobrunbigqueryloader/uploader.go

```diff
@@ -197,12 +195,13 @@ func (o *jobLoaderOptions) Run(ctx context.Context) error {
 		}
 
 		currentUploaders.Add(1)
+		jrID := jobRunID
 		go func() {
```

Comment:
that will work, but what do you think about making it a parameter on the go func instead of a closure?

Reply:
changed